src/views/Stack/AnimatedScreen.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { NativeScreen } from 'react-native-screens';
+import Animated from 'react-native-reanimated';
+import memoize from '../../utils/memoize';
+
+type Props = ViewProps & {
+  next?: Animated.Node<number>;
+  children: React.ReactNode;
+};
+
+const { cond, eq, call, onChange } = Animated;
+
+export default class AnimatedScreen extends React.Component<Props> {
+  componentDidUpdate(prevProps: Props) {
+    if (prevProps.next && !this.props.next) {
+      // If there's no next screen, this is the last one, keep it active
+      this.setActive(true);
+    }
+  }
+
+  private screen = React.createRef<any>();
+
+  private subscribe = memoize((next: Animated.Node<number>) =>
+    onChange(
+      cond(eq(next, 1), 0, 1),
+      call([next], ([value]: readonly number[]) => {
+        // If the next screen is transitioning, i.e. its progress is less than 1, set this screen to active
```

Comment:
Have you verified it's actually invoked on begging of transition and cancelling it?

Reply:
If you check the modal presentation iOS example, the previous screen is visible underneath. In this case, you can see that the previous screen disappears when the transition finishes, when appears when you start the transition by gesture/pressing back button.